### ISSUE TITLE
ADF v2.0.4: python "aws-cdk-" submodules have conflicting dependencies

### ISSUE THREAD

**Issue by at7925** [awslabs/aws-deployment-framework#166]
The problem starts happening in us-east-1 on 23-oct-2019. Errors from CreateOrUpdatePipelines log:
```
[Container] 2019/10/23 13:21:42 Running command npm install cdk@1.13.1 -g -y --quiet --no-progress 
/usr/local/bin/cdk -> /usr/local/lib/node_modules/cdk/bin/cdk 
> core-js@2.6.10 postinstall /usr/local/lib/node_modules/cdk/node_modules/core-js 
> node postinstall || echo "ignore"  
+ cdk@1.13.1 
added 238 packages from 218 contributors in 10.87s 
[Container] 2019/10/23 13:21:53 Running command aws s3 cp s3://$SHARED_MODULES_BUCKET/adf-build/ ./adf-build/ --recursive --quiet 
[Container] 2019/10/23 13:21:58 Running command pip install -r adf-build/requirements.txt -q -t ./adf-build 
ERROR: aws-cdk-aws-cloudwatch 1.14.0 has requirement aws-cdk.aws-iam>=1.14.0,~=1.14, but you'll have aws-cdk-aws-iam 1.13.1 which is incompatible. 
ERROR: aws-cdk-aws-cloudwatch 1.14.0 has requirement aws-cdk.core>=1.14.0,~=1.14, but you'll have aws-cdk-core 1.13.1 which is incompatible. 
ERROR: aws-cdk-aws-autoscaling-common 1.14.0 has requirement aws-cdk.aws-iam>=1.14.0,~=1.14, but you'll have aws-cdk-aws-iam 1.13.1 which is incompatible. 
ERROR: aws-cdk-aws-autoscaling-common 1.14.0 has requirement aws-cdk.core>=1.14.0,~=1.14, but you'll have aws-cdk-core 1.13.1 which is incompatible. 
ERROR: aws-cdk-aws-applicationautoscaling 1.14.0 has requirement aws-cdk.aws-iam>=1.14.0,~=1.14, but you'll have aws-cdk-aws-iam 1.13.1 which is incompatible. 
ERROR: aws-cdk-aws-applicationautoscaling 1.14.0 has requirement aws-cdk.core>=1.14.0,~=1.14, but you'll have aws-cdk-core 1.13.1 which is incompatible. 
. . .
```

**Comment by bundyfx** [comment]
Hi @at7925 

Yes, we're locking only a few cdk modules to 1.13.1 however there are other ones being installed indirectly that are not locked. This should have no effect on the outcome of the pipeline but I will make a patch release in the next 24 hours that locks all those indirectly installed also.

Thanks!

**Comment by at7925** [comment]
Confirmation: solved in ADF v2.1.0
